### ISSUE TITLE
Edges appending to only one instance of Svelvet container

### ISSUE THREAD

**Issue by gonzaloaune** [open-source-labs/Svelvet#513]
### Description

If you have more than one Svelvet app container and you have Edges on your Nodes, the Edges are only appended to the first instance of the Svelvet app due to this line https://github.com/open-source-labs/Svelvet/blob/bd69dc3c6b0393569fe62caa766928c90065905a/src/lib/components/Edge/Edge.svelte#L20

### Reproduction

https://stackblitz.com/edit/vitejs-vite-3df9v8?file=package.json,src%2FApp.svelte

1) Create 2 instances of Svelvet apps with Edges, Anchors and Nodes
2) See the first one will have double the Edges
3) See the subsequent instances of Svelvet apps not drawing the Edges

### System information

    svelte: ^4.2.12 => 4.2.12
    svelte-loader: ^3.2.0 => 3.2.0
    svelvet: 10.0.2 => 10.0.2

### 👨‍👧‍👦 Contributing

- [ ] 🙋‍♂️ Yes, I'd love to make a PR to fix this bug!

**Comment by Khaalid-oh** [comment]
Hi @gonzaloaune I can help with this

**Comment by Khaalid-oh** [comment]
I can't seem to reach you on sphinx chat

**Comment by turizspace** [comment]
#515

**Comment by aliraza556** [comment]
HI @gonzaloaune, this issue is avaliable then please assign me?

**Comment by caydyan** [comment]
Submitted a fix in https://github.com/open-source-labs/Svelvet/pull/551 for Sphinx bounty #2284.

The PR scopes edge reparenting to the nearest `.svelvet-graph-wrapper`, adds jsdom regression coverage for multiple graph wrappers, and lists the validation run.

Payout address: `bc1qev5ant33v5y89qqjvcf4mh9hlax5svqf5xd7gc`.

**Comment by Khaalid-oh** [comment]
Are Sphinx Bounties still active?

**Comment by turizspace** [comment]
i don't think so @Khaalid-oh  the community platform been ded for a while now.
